### PR TITLE
[5.0] Add parameter for setting template link color in dark mode

### PR DESCRIPTION
**Pull request by wilsonge**
Fixes the accessibility of the links when in dark mode.

### Summary of Changes
Adds a parameter to control the link color when in dark mode (this is a different color from the current one which doesn't meet a11y requirements).

I'll happily take any advice on how best to word both the label for the field and maybe even adding an option so it's clear. With us already talking "dark hue" and "dark text" it's really not that clear to the user what is going on (in my opinion)

### Testing Instructions
Template link now accessible in dark mode to AA and controllable by the user in the template styles. 

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
lgtm

**Comment by wilsonge** [comment]
OK I'll open it up then. Somehow I'm not sure this will be intuitive like I said. But better to at least allow a11y compliance

File: administrator/language/en-GB/tpl_atum.ini [hunk 1]
```diff
@@ -10,6 +10,7 @@ TPL_ATUM_COLORS_HUE="Choose your hue value for the dark template colour"
 TPL_ATUM_COLORS_SETTINGS_BG_LIGHT_LABEL="Light Background"
 TPL_ATUM_COLORS_SETTINGS_LABEL="Colour Settings"
 TPL_ATUM_COLORS_SETTINGS_LINK_COLOR_LABEL="Link Colour"
+TPL_ATUM_COLORS_SETTINGS_LINK_COLOR_DARK_LABEL="Link Colour (Dark Mode)"
```
Comment (Quy) [inline administrator/language/en-GB/tpl_atum.ini]:
```suggestion
TPL_ATUM_COLORS_SETTINGS_LINK_COLOR_DARK_LABEL="Link Colour (Dark Mode)"
TPL_ATUM_COLORS_SETTINGS_LINK_COLOR_LABEL="Link Colour"
```

**Comment by HLeithner** [comment]
thanks

**Pull request merged**